tests/integration/test_snakemake.py

```diff
@@ -33,3 +33,46 @@ def test_missing_results(test_specific_results_dir, mocker, caplog):
         )
     assert exit.value.code == 1
     assert "MissingOutputException" in caplog.text
+
+
+@pytest.mark.slow
+@pytest.mark.parametrize("outputs_specified", [True, False])
```

Comment:
this is the test @zmbc originally wrote. I added the `outputs_specified` parametrization